### PR TITLE
docs(tutorial): servless manage instances

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+tags: serverless instances python
+categories:
+  - compute
+  - serverless
+dates:
+  validation: 2021-12-21
+  posted: 2021-12-21
+---
+
+In this tutorial you will discover some examples of Instances automation using Python and the serverless framework:
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:17]:
```suggestion
In this tutorial you will discover some examples of Instance automation using Python and the serverless framework:
```

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+  - You have [generated an API key](/console/my-project/how-to/generate-api-key/)
+  - You have Python pip installed on your machine
+
+</Message>
+
+## Automatically backup your Instance
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:28]:
```suggestion
## Automatically back up your Instance 
```

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+  - You have [generated an API key](/console/my-project/how-to/generate-api-key/)
+  - You have Python pip installed on your machine
+
+</Message>
+
+## Automatically backup your Instance 
+
+For this example we will use the Scaleway SDK and the serverless framework, for more information please refer to Python SDK Documentation and the [how-to on serverless framework](/compute/functions/api-cli/fun-uploading-with-serverless-framework/)
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:30]:
```suggestion
For this example we will use the Scaleway SDK and the serverless framework. For more information please refer to the Python SDK Documentation and the [how-to on serverless framework](/compute/functions/api-cli/fun-uploading-with-serverless-framework/)
```

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+1. Create a project using the serverless framework (in a terminal):
+    ```
+    serverless create --template-url https://github.com/scaleway/serverless-scaleway-functions/tree/master/examples/python3 --path myService
+    ```
+
+2. Install the Scaleway SDK using `pip`:
+    ```
+    pip install scaleway-sdk --target ./package
+    ```
+3. Edit the fily `handler.py` in a text editor. Then save the file and exit the editor.
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:41]:
```suggestion
3. Edit the file `handler.py` in a text editor as follows. Then save the file and exit the editor. 
```

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+    env:
+        API_KEY: {Your API KEY} 
+        PROJECT_ID: {Your project Id}
+        SNAPSHOT_NAME: {Name of the snapshot}
+        VOLUME_ID: {Volume to be backup}
+    ```
+
+5. Deploy using `serverless deploy`.
+
+## Automatically shut down / start instances
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:84]:
```suggestion
## Automatically shut down / start Instances
```

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+For this example we will use the native python library `urllib`, which enables us to write the code down directly into the Scaleway console.
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:86]:
```suggestion
For this example we will use the native python library `urllib`, which enables us to write the code directly into the Scaleway console.
```

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+All information about the Instance API can be found in the [developers documentation](https://developers.scaleway.com/en/products/instance/api/#get-2c1c6f)
+
+<Message type="note">
+
+You require your [Instance information] for the following steps.
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:91]: missing link?

**Review by ldecarvalho-doc — APPROVED** [review]

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 8]
```diff
@@ -0,0 +1,147 @@
+---
+meta:
+  title: Managing compute Instances with the serverless framework
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:3]: Could the title just be "Managing Instances with the serverless framework"? or "Scaleway Instances"?

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+            res=e.read().decode()
+            
+        return {
+            "body": json.loads(res),
+            "statusCode": 200,
+        }
+    ```
+
+3. Add the environment variable `X-AUTH-TOKEN`.
+4. Create cron jobs dedicated to start or stop your instance in the **Advanced Section**.
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:131]:
```suggestion
4. Create cron jobs dedicated to start or stop your Instance in the **Advanced Section**.
```

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+5. Deploy using `serverless deploy`.
+
+## Automatically shut down / start Instances
+
+For this example we will use the native python library `urllib`, which enables us to write the code directly into the Scaleway console.
+All information about the Instance API can be found in the [developers documentation](https://developers.scaleway.com/en/products/instance/api/#get-2c1c6f)
+
+<Message type="note">
+
+You require your [Instance information] for the following steps.
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:91]: so is this bit not meant to have a link?

File: tutorials/manage-instances-with-serverless-framework/index.mdx (hunk 8 above)
Comment (bene2k1) [reply tutorials/manage-instances-with-serverless-framework/index.mdx:3]:
```suggestion
  title: Managing Scaleway Instances with the serverless framework
```

File: tutorials/manage-instances-with-serverless-framework/index.mdx (hunk 10 above)
Comment (bene2k1) [reply tutorials/manage-instances-with-serverless-framework/index.mdx:91]:
```suggestion
You require your [Instance information](https://developers.scaleway.com/en/products/instance/api/#get-2c1c6f) for the following steps.
```

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+---
+
+In this tutorial you will discover some examples of Instance automation using Python and the serverless framework:
+
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+  - You have [generated an API key](/console/my-project/how-to/generate-api-key/)
+  - You have Python pip installed on your machine
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:24]:
```suggestion
  - You have [pip](https://pypi.org/project/pip/) installed on your machine
```

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+4. Create cron jobs dedicated to start or stop your Instance in the **Advanced Section**.
+    * To **stop** an Instance:
+    ```yaml
+    {
+      "zone":"{ZONE}",
+      "server_id":"{XXXXXXX-XXXX-XXX-XXXX-XXXXXXX}",
+      "action":"poweroff"
+    }
+    ```
+    * To **start** an instance:
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:140]:
```suggestion
    * To **start** an Instance:
```

**Review by RoRoJ — APPROVED** [review]

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+---
+
+In this tutorial you will discover some examples of Instance automation using Python and the serverless framework:
+
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+  - You have [generated an API key](/console/my-project/how-to/generate-api-key/)
+  - You have [pip](https://pypi.org/project/pip/) installed on your machine
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:24]: Maybe also add requirements to install npm and serverless, as described in /compute/functions/api-cli/fun-uploading-with-serverless-framework/ ?

File: tutorials/manage-instances-with-serverless-framework/index.mdx [hunk 14]
```diff
@@ -0,0 +1,147 @@
...
+1. Create a project using the serverless framework (in a terminal):
+    ```
+    serverless create --template-url https://github.com/scaleway/serverless-scaleway-functions/tree/master/examples/python3 --path myService
+    ```
+
+2. Install the Scaleway SDK using `pip`:
+    ```
+    pip install scaleway-sdk --target ./package
+    ```
+3. Edit the file `handler.py` in a text editor as follows. Then save the file and exit the editor.
```
Comment (RoRoJ) [inline tutorials/manage-instances-with-serverless-framework/index.mdx:41]:
```suggestion
3. Navigate to the `myService` directory and edit the file `handler.py` in a text editor as follows. Then save the file and exit the editor. 
```

**Review by lucasscw — APPROVED** [review]

**Review by ofranc — APPROVED** [review]

**Pull request merged**